### PR TITLE
[CROW-242] Parse pledged value

### PR DESCRIPTION
**Pull request by FacuEM**
#### :page_facing_up: Description:

This PR presents a solution to the problem in the pledge, in which the amount was not being sent with the correct units. For this a function called `parseUnits` is created that makes use of the `bignumber.js` library to convert the pledge value to the correct one. Also some adjustments are made since the backend response for a campaign has changed.

---

#### :pushpin: Notes:

* The currentAmount field is not changing when pledging, however the pledge counter is changing correctly.

### CODE REVIEW

File: components/ProjectSideBar/projectSideBar.js [hunk 1]
```diff
@@ -18,12 +19,15 @@ import {
 
 export const ProjectSideBar = ({ campaign, onClick, isOwner, setIsPledge }) => {
   const router = useRouter();
-  const { goal, currentAmount } = campaign;
+  const { isUserAuthenticated } = useAuth();
+  const { goal, currentAmount } = campaign.campaign;
```
Comment (renacargnelutti) [inline components/ProjectSideBar/projectSideBar.js:23]: NIT: `campaign.campaign` looks odd. Probably we can improve it in the future

File: components/ProjectSideBar/projectSideBar.js [hunk 2]
```diff
@@ -58,12 +62,12 @@ export const ProjectSideBar = ({ campaign, onClick, isOwner, setIsPledge }) => {
       <BoxContainer>
         <Box>
           <Subtitle>Revenue</Subtitle>
-          <Value>{currentAmount[0].amount}USDT</Value>
+          <Value>{currentAmount[0].amount} USDT</Value>
```
Comment (renacargnelutti) [inline components/ProjectSideBar/projectSideBar.js:65]: NIT: As we talked with @lucasgalarce the token symbol should be dynamic. We can improve it in the future

File: pages/project/[id].page.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ const ProjectPage = () => {
     watch(PLEDGE_AMOUNT)
   );
 
-  const isOwner = address === campaign?.owner?.publicAddress;
+  const isOwner = address === campaign?.campaign?.owner?.publicAddress;
```
Comment (renacargnelutti) [inline pages/project/[id].page.js:56]: @lucasgalarce , did we change the response for that endpoint that was returning `{ data: { campaign: { campaign }, pledges }}`?

Comment (lucasgalarce) [reply pages/project/[id].page.js:56]: Yes, is as Rena said. I guess that the latest version of api isn't deployed. I'm deploying again now

**Review by renacargnelutti — APPROVED** [review]
LGTM 💪

**Pull request merged**